client/v3/client.go

```diff
@@ -376,6 +384,10 @@ func newClient(cfg *Config) (*Client, error) {
 		creds = credentials.NewTransportCredential(cfg.TLS)
 	}
 
+	if cfg.Token != "" && (cfg.Username != "" || cfg.Password != "") {
+		return nil, errors.New("Username/Password and Token configurations are mutually exclusive")
```

Comment:
Minor observation/question: thinking about the work we have been doing with errors, does setting this error as an exported variable make sense? Would it be valuable? WDYT @ahrtr?

Reply:
Sounds like a good suggestion.

Reply:
Given the number of follow-ups we have for this pull request, and the time it has been open, I think it's okay to do this in a follow-up PR.